manager/src/main/java/com/redhat/service/bridge/manager/connectors/creation/CreateConnectorWorker.java

```diff
@@ -95,4 +104,23 @@ protected ConnectorEntity updateEntityForError(ConnectorEntity connectorEntity,
         connectorEntity.setStatus(ConnectorStatus.FAILED);
         return connectorEntity;
     }
+
+    @Override
+    protected void afterSuccessfullyUpdated(ConnectorEntity c) {
+        //Check that the Managed Connector has been created
+        eventBus.request(Events.MANAGED_CONNECTOR_CREATED_EVENT, c)
+                .onFailure()
+                .retry()
+                .withBackOff(DEFAULT_BACKOFF)
+                .withJitter(DEFAULT_JITTER)
+                .atMost(MAX_RETRIES)
+                .subscribe().with(
+                        success -> {
+                            /* NOOP */},
+                        failure -> {
+                            BridgeError error = bridgeErrors.findByException(ConnectorCreationException.class);
```

Comment:
I don't quite understand the whole `BridgeError` thing.. 

I look up an exception to get it's reason.. which seems weird.

Reply:
It didn't seem to work (when writing tests).. 

So I've refactored it to now rely on `ConnectorsOrchestrator` as is the case for other state re-tries.